README.md

```diff
@@ -32,30 +32,39 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. What is the DOM?
 
+    (Document Object Model) A bridge between the content and the browser.
+
 2. What is an event?
 
+    Any detectable interaction with a webpage.
+
 3. What is an event listener?
 
+    A method that listens for specific events and produces a callback triggered by the event.
+
 4. Why would we convert a NodeList into an Array?
 
+    With an array you can utilize the ".forEach()" method to loop through data.
```

Comment:
NodeLists actually already contain the forEach method - converting to an Array will give us the rest of the array methods